sematic/ui/packages/main/src/runs/RunIndex.tsx

```diff
@@ -144,8 +144,10 @@ const TableColumns: Array<RunListColumn> = [
 ];
 
 export function RunIndex() {
+    const params = new URLSearchParams(document.location.search);
```

Comment:
The deeplink information should be in the hash instead of the query string(search). Hashes are supposed to store page state. All other deeplinks are using hash. Querystrings can be read by both Frontend and Backend and are reserved for other features, eg, feature flags.

You can use `useLocation()` to get the hash.